ui/oopsyraidsy/data/00-misc/general.ts

```diff
@@ -113,6 +163,62 @@ const triggerSet: OopsyTriggerSet<Data> = {
           delete data.raiseTracker[matches.targetId];
       },
     },
+    {
+      id: 'General Overwritten Boss Mit',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: targetMitAbilityIds.concat(partyMitAbilityIds) }),
+      mistake: (data, matches) => {
+        const isTargetMit = targetMitAbilityIds.includes(matches.id);
+        const isPartyMit = partyMitAbilityIds.includes(matches.id);
+        if (isTargetMit && matches.targetId === 'E0000000') // missed
+          return;
+        if (isPartyMit && !data.party.partyIds_.includes(matches.sourceId))
+          return;
+        if (isPartyMit && matches.targetId !== matches.sourceId)
+          return;
+        data.mitTracker ??= {};
+        const target = isTargetMit ? matches.targetId : 'party';
+        const lastTime = data.mitTracker[target]?.[matches.id]?.time;
+        const lastSource = data.mitTracker[target]?.[matches.id]?.source;
+        const newTime = new Date(matches.timestamp).getTime();
+        const newSource = data.ShortName(matches.source);
+        const duration = isTargetMit
+          ? targetMitAbilityIdToDuration[matches.id]
+          : partyMitAbilityIdToDuration[matches.id];
+        if (lastTime !== undefined && lastSource !== undefined && duration !== undefined) {
+          const diff = newTime - lastTime;
+          const leeway = (duration * 1000 - diff) > Options.MinimumTimeForOverwrittenMit * 1000;
```

Comment:
I think you mean `data.options` here, rather than importing this and checking the default set of option values.